app/views/candidate_mailer/reinstated_offer.text.erb

```diff
@@ -4,7 +4,7 @@ Dear <%= @application_form.first_name %>
 
 The course starts in <%= @course_option.course.start_date.to_fs(:month_and_year) %>.
 
-<% if @application_choice.offer.conditions.where(status: 'met').count.zero? %>
+<% if @application_choice.offer.conditions.any? { |c| c.status != 'met' } %>
```

Comment:
a bit of neat-freakery: 

```
<% if @application_choice.offer.conditions.all?(&:met?) %>
```

Reply:
on the fine line between code and ascii art

Reply:
```
> [].all?(&:met?)
=> true
```
huh

Reply:
@carlosmartinez Turns out this was wrong :(  You inverted the behaviour.